### ISSUE TITLE
Cifar10 example with VGG: accuracy stuck at 0.087

### ISSUE THREAD

**Issue by iuliancioarca** [FluxML/model-zoo#220]
I ran the stock `cifar10.jl` but the accuracy is stuck at 0.087. I visually checked a few of the train and test data labels and they match with the pictures.
Has anyone else encountered this?

**Comment by AdarshKumar712** [comment]
Yes I also faced the same problem earlier. Though I am not completely sure, I think the problem is with the learning rate. It should be smaller as you can verify the loss is increasing. The model seems to be diverging.
Other than that, you can also give a try to the updated cifar10.jl model #210. It was working fine for me.
